src/IFX007T-Motor-Control.h

```diff
@@ -0,0 +1,87 @@
+
+#ifndef IFX007T_MOTOR_CONTROL_H
+#define IFX007T_MOTOR_CONTROL_H
+
+#include <Arduino.h>
+
+//======= Very Important ========
+//#define DEBUG_IFX007T         //Uncomment, if you wish debug output or tune the motor (Disables automatic V_neutralOffset)
+//===============================
+
+#ifdef DEBUG_IFX007T
```

Comment:
ah! you already implenented conditional debug traces 👯‍♀️   then you do not need to comment the debug macro traces in your .cpp or?

Reply:
Is it still working fine with the Debug functions? Because I thought you mentioned something that the serial.prints of the functions are disturbing the commutation?